packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -667,7 +667,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             tryFetchBlob<[string, string][]>(aliasBlobName),
         ]);
 
-        const loadExisting = existing === true || context.existing === true;
+        const loadExisting = existing === true;
```

Comment:
Please experiment with making this parameter required (it also needs a comment update as we won't be checking the context's flag anymore)

Reply:
This is the comment that needs to be updated:
```
* @param existing - (optional) When loading from an existing snapshot. Precedes context.existing if provided
```

IMO you don't need to add a comment here about "no longer checking context".

Did you try making the function parameter required on this function?